presto-main/src/test/java/com/facebook/presto/operator/scalar/TestGroupingOperationFunction.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.operator.scalar;
+
+import org.testng.annotations.Test;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static com.facebook.presto.operator.scalar.GroupingOperationFunction.grouping;
+import static org.testng.Assert.assertEquals;
+
+public class TestGroupingOperationFunction
+{
+    @Test
```

Comment:
Done. Added a similar test to `AbstractTestQueries` to test that we correctly pick the bigint implementation when appropriate.